beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -6777,6 +6777,27 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         self.data_availability_checker.data_availability_boundary()
     }
 
+    /// Returns true if epoch is within the data availability boundary
+    pub fn is_within_data_availability_boundary(&self, epoch: Epoch) -> bool {
```

Comment:
We do already have a method, we could pass through here `self.data_availability_checker.da_check_required_for_epoch()`

Reply:
Switched to `da_check_required_for_epoch` in 96e2363d721f55d79f66f98f15fd61a854c7dc62